src/components/editPostModal/index.js

```diff
@@ -6,10 +6,11 @@ import { patch } from '../../service/apiClient'
 
 
 // passing current post ID and message content through props
-const EditPostModal = ({ id, content, setTriggerUpdate}) => {
+const EditPostModal = ({ id, content, setTriggerUpdate, name}) => {
     const { closeModal } = useModal()
     const [message, setMessage] = useState(null)
     const [text, setText] = useState(content)
+    const initials = name[0] + name.split(' ')[1][0]
```

Comment:
This is dangerous to do without checking the length of name/ the length of the split array. Add some checks before doing this to avoid getting an undefined error.